### PR TITLE
Updating screenshots and instructions for Dynamic Vegetation user guide

### PR DESCRIPTION
**Pull request by jckand-amzn**
Signed-off-by: jckand-amzn <82226555+jckand-amzn@users.noreply.github.com>

## Change summary

Updating screenshots/requirements/workflow steps/component names for Dynamic Vegetation user guide documentation.

### Submission Checklist:

* [X ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [X ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [X ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [X ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 1]
```diff
@@ -6,7 +6,27 @@ weight: 100
...
+
+
+**Pre-requisites/Setup**
+
+* Vegetation assets spawned via a Vegetation Layer Spawner component require a surface to plant on. This can be done one of several ways:
+   * In the following workflow, we established a surface by adding a **Mesh Surface Tag Emitter** component to the *Ground* entity that exists as a part of the *Atom Default Environment* entity hierarchy that is added to each new level.
+   * You can also establish a surface by:
+     - Adding a **Mesh Surface Tag Emitter** component to any other entity with a Mesh component
+     - Adding a **Shape Surface Tag Emitter** component to an entity with a Shape component
+     - Adding [Terrain](/docs/user-guide/components/reference/terrain/layer_spawner) to the level
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:19]: Let's update this to use the active voice. Using the past tense to refer to a future step is a potential source of confusion. Also, make sure to end bulleted list items with a period.
```suggestion
To spawn vegetation assets using a Vegetation Layer Spawner component, you must have a surface to plant on. You can establish a surface by doing any of the following:

* Add a **Mesh Surface Tag Emitter** component to the *Ground* entity that exists as a part of the *Atom Default Environment* entity hierarchy that is included with each new level. (This technique is used in the example that follows.)
* Add a **Mesh Surface Tag Emitter** component to any other entity that has a Mesh component.
* Add a **Shape Surface Tag Emitter** component to an entity that has a Shape component.
* Add [Terrain](/docs/user-guide/components/reference/terrain/layer_spawner) to the level.
```

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 2]
```diff
@@ -6,7 +6,27 @@ weight: 100
...
+      
+* Vegetation assets exist to spawn:
+   * In the following workflow, we created a saved prefab to spawn via the **Vegetation Layer Spawner** by:
+      1. Creating a new entity
+      2. Adding a **Mesh** component
+      3. Assigning a vegetation `.fbx` file to the **Mesh** component
+      4. Naming and saving the configured entity as a prefab
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:26]:
```suggestion
You must also have vegetation assets that the **Vegetation Layer Spawner** can spawn. For the example that follows, we created a prefab named `DryGrassLarge.prefab` using the following steps:
1. Create a new entity.
2. Add a **Mesh** component.
3. Assign a vegetation `.fbx` file to the **Mesh** component.
4. Save the entity as a prefab.
```

**Review by willihay — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 3]
```diff
@@ -22,35 +42,35 @@ Creating a vegetation layer is the first and most basic step in creating your dy
...
 
-1. Select **TextBox**, click **Add Component**, and select the **Box Shape** component.
+1. Select *TestBox*, click **Add Component**, and select the **Box Shape** component.
 
 1. Adjust the size and position of the shape so that it's large enough for your purposes and intersects with the ground.
 
     ![Adjust your shape to cover a sufficient area and intersect with the ground.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-adjust-shape.png)
 
-1. Select **BasicCoverage** and, in the **Vegetation Reference Shape** component, click the target symbol and select the **TestBox** entity.
+1. Select *BasicCoverage* and, in the **Shape Reference** component, click the target symbol and select the *TestBox* entity.
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:59]: We can add inline icons for interface elements.

```suggestion
1. Select *BasicCoverage* and, in the **Shape Reference** component, click the {{< icon "picker.svg" >}} **entity picker** icon and select the *TestBox* entity.
```

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 4]
```diff
@@ -22,35 +42,35 @@ Creating a vegetation layer is the first and most basic step in creating your dy
...
 
 1. Click **Add Required Component** and choose **Vegetation Asset List**.
 
     The **Vegetation Asset List** component defines what to plant. This is where you specify vegetation assets.
 
     ![On the Vegetation Layer Spawner, click Add Component and select Vegetation Asset List.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-asset-list.png)
 
-1. In the **Vegetation Asset List** component, next to **Mesh Asset**, click **Browse (...)**.
+1. In the **Vegetation Asset List** component, change the **Instance Spawner** type to Prefab, and then next to **Prefab Asset**, click the folder icon to browse for your asset.
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:67]:
```suggestion
1. In the **Vegetation Asset List** component, change the **Instance Spawner** type to Prefab, and then next to **Prefab Asset**, click the {{< icon "file-folder.svg" >}} **folder** icon to browse for your asset.
```

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 5]
```diff
@@ -22,35 +42,35 @@ Creating a vegetation layer is the first and most basic step in creating your dy
...
 
-    ![Click Browse (…) to select a Mesh Asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-browse.png)
+    ![Click the Folder icon to select a Prefab Asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-browse.png)
 
-1. In the **Search** bar, enter **grass** and select one of the grass assets in the results.
+1. We have a saved prefab with *grass* in the name, so in the **Search** bar, we enter *grass* and select the grass asset in the results.
 
-    ![Click Browse (…) to select a Mesh Asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-asset-grass.png)
+    ![Click the Folder icon to select a Prefab Asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-asset-grass.png)
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:73]:
```suggestion
![Use search to find a prefab asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-asset-grass.png)
```

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 6]
```diff
@@ -22,35 +42,35 @@ Creating a vegetation layer is the first and most basic step in creating your dy
...
     ![On the Vegetation Layer Spawner, click Add Component and select Vegetation Asset List.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-asset-list.png)
 
-1. In the **Vegetation Asset List** component, next to **Mesh Asset**, click **Browse (...)**.
+1. In the **Vegetation Asset List** component, change the **Instance Spawner** type to Prefab, and then next to **Prefab Asset**, click the folder icon to browse for your asset.
 
-    ![Click Browse (…) to select a Mesh Asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-browse.png)
+    ![Click the Folder icon to select a Prefab Asset.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-browse.png)
 
-1. In the **Search** bar, enter **grass** and select one of the grass assets in the results.
+1. We have a saved prefab with *grass* in the name, so in the **Search** bar, we enter *grass* and select the grass asset in the results.
```
Comment (micronAMZN) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:71]:
```suggestion
1. In the **Pick a Prefab** window, enter *grass* into the search bar to filter the list of available prefabs. Select the `DryGrassLarge.prefab` asset in the results.
```

**Review by micronAMZN — CHANGES_REQUESTED** [review]
A couple of minor suggestions to add interface icons and clarify a step.

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 7]
```diff
@@ -6,7 +6,27 @@ weight: 100
 toc: true
 ---
 
-Creating a vegetation layer is the first and most basic step in creating your dynamic vegetation. The following procedure uses a simple workflow and assets from the Starter Game project.
+Creating a vegetation layer is the most basic step in creating your dynamic vegetation, however a few requirements must be met in order to spawn vegetation successfully.
+
+
+**Pre-requisites/Setup**
```
[the remark this answers is not in the document]
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Fixed.

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md (hunk 1 above)
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Updated with suggested change.

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md (hunk 2 above)
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Updated with suggested change.

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md (hunk 3 above)
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Sweet, updated!

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md (hunk 4 above)
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Updated!

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md (hunk 5 above)
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Updated!

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md (hunk 6 above)
Comment (jckand-amzn) [reply content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md]: Updated using suggested change.

**Review by micronAMZN — APPROVED** [review]

File: content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md [hunk 8]
```diff
@@ -16,42 +36,42 @@ Creating a vegetation layer is the first and most basic step in creating your dy
...
 
-1. Click **Add Required Component** and choose **Vegetation Asset List**.
+8. Click **Add Required Component** and choose **Vegetation Asset List**.
 
     The **Vegetation Asset List** component defines what to plant. This is where you specify vegetation assets.
 
     ![On the Vegetation Layer Spawner, click Add Component and select Vegetation Asset List.](/images/user-guide/vegetation/dynamic/create-vegetation-layer-asset-list.png)
 
-1. In the **Vegetation Asset List** component, next to **Mesh Asset**, click **Browse (...)**.
+9. 1. In the **Vegetation Asset List** component, change the **Instance Spawner** type to Prefab, and then next to **Prefab Asset**, click the {{< icon "file-folder.svg" >}} **folder** icon to browse for your asset.
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/environment/vegetation/procedures/layer.md:67]:
```suggestion
9. In the **Vegetation Asset List** component, change the **Instance Spawner** type to Prefab, and then next to **Prefab Asset**, click the {{< icon "file-folder.svg" >}} **folder** icon to browse for your asset.
```

**Review by willihay — APPROVED** [review]

**Pull request merged**